src/components/ActionBar.ts

```diff
@@ -0,0 +1,39 @@
+import { ReactElement } from 'react';
+import { div, h } from 'react-hyperscript-helpers';
+import { ButtonPrimary } from 'src/components/common';
+
+type ActionBarProps = {
+  /** The prompt to display next to the action bar button. */
+  prompt: string | ReactElement;
+  /** The action bar button's label. */
+  actionText: string | ReactElement;
+  /** The onClick handler for the action bar button.  */
+  onClick: () => void;
```

Comment:
Should the function signature take in a MouseEvent?

Reply:
I don't think so. We don't have a use for it today, and `onClick` has no type downstream so typescript doesn't require it.